TOC.md

```diff
@@ -37,6 +38,7 @@
   + 监控
     - [概述](dev/how-to/monitor/overview.md)
     - [监控 TiDB 集群](dev/how-to/monitor/monitor-a-cluster.md)
+    - [TiDB Binlog 集群监控](dev/how-to/monitor/tidb-binlog-monitor.md)
```

Comment:
```suggestion
    - [TiDB Binlog 集群监控](dev/how-to/monitor/tidb-binlog.md)
```

Reply:
Suggest to use ` tidb-binlog.md ` as the file name in different how-to parts?